integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/DatabaseCreateIfNotExistsAction.java

```diff
@@ -6,28 +6,28 @@
 import org.slf4j.LoggerFactory;
 
 import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
-import com.sequenceiq.it.cloudbreak.newway.RdsConfigEntity;
+import com.sequenceiq.it.cloudbreak.newway.entity.DatabaseEntity;
 import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
 
-public class RdsConfigCreateIfNotExistsAction implements Action<RdsConfigEntity> {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RdsConfigCreateIfNotExistsAction.class);
+public class DatabaseCreateIfNotExistsAction implements Action<DatabaseEntity> {
```

Comment:
please create a database package inside action and move database related actions there